src/widgets/category/title/search-title-view.js

```diff
@@ -22,7 +22,7 @@ module.exports = cdb.core.View.extend({
   },
 
   initialize: function () {
-    this.widgetModel = this.options.widgetModel;
+    this.model = this.options.widgetModel;
```

Comment:
Isn't this a bit sacrificing clarity for convention, where we don't really need to follow a convention? I find now myself constantly forgetting whether `this.model` refers to the dataview or the widget.